plugins/woocommerce/src/Internal/Admin/Onboarding/OnboardingSetupWizard.php

```diff
@@ -223,14 +223,9 @@ public function component_settings( $settings ) {
 			return $settings;
 		}
 
-		include_once WC_ABSPATH . 'includes/admin/helper/class-wc-helper-options.php';
-		$wccom_auth                 = \WC_Helper_Options::get( 'auth' );
-		$profile['wccom_connected'] = empty( $wccom_auth['access_token'] ) ? false : true;
-
-		$settings['onboarding']['currencySymbols'] = get_woocommerce_currency_symbols();
```

Comment:
Probably need to rebase on the other branch because this removal breaks the store onboarding so I have to manually set the currency settings to test this PR

Reply:
@rjchow Do you mean that this PR doesn't work unless the value is set manually? I tested it on a JN site and didn't encounter any errors. 🤔

Reply:
As in setting the country in core profiler doesn’t set the currency etc, on a JN live branch build - so I tested it by setting them via the wc settings page

Reply:
@rjchow I see what you mean now. I'll rebase.